src/Controller/SantaController.php

```diff
@@ -317,6 +325,7 @@ public function cancel(Request $request, string $application): Response
         return $this->redirectToRoute('homepage');
     }
 
+    #[Route('/spoil', name:'spoil', methods:['GET'])]
```

Comment:
The method POST is missing here, isn't it?

Reply:
You're right it was. I added it, thank you